tests/pipelines/photon/test_pipeline_photon.py

```diff
@@ -0,0 +1,258 @@
+import unittest
+
+import numpy as np
+import torch
+from transformers import AutoTokenizer
+from transformers.models.t5gemma.configuration_t5gemma import T5GemmaConfig, T5GemmaModuleConfig
+from transformers.models.t5gemma.modeling_t5gemma import T5GemmaEncoder
+
+from diffusers.models import AutoencoderDC, AutoencoderKL
+from diffusers.models.transformers.transformer_photon import PhotonTransformer2DModel
+from diffusers.pipelines.photon.pipeline_photon import PhotonPipeline
+from diffusers.schedulers import FlowMatchEulerDiscreteScheduler
+
+from ..pipeline_params import TEXT_TO_IMAGE_PARAMS
+from ..test_pipelines_common import PipelineTesterMixin
+
+
+class PhotonPipelineFastTests(PipelineTesterMixin, unittest.TestCase):
```

Comment:
```suggestion
@pytest.mark.xfail(condition=is_transformers_version(">", "4.57.1"), reason="transformers RoPE config refactor", strict=True)
class PhotonPipelineFastTests(PipelineTesterMixin, unittest.TestCase):
```
Following https://github.com/huggingface/diffusers/pull/12456#issuecomment-3424228544, https://github.com/huggingface/diffusers/pull/12456#issuecomment-3424237711. @sayakpaul, since the code that would trigger the failure is in `get_dummy_components`, lots of tests will fail with the new change, so I think it is best to decorate the test class rather than specific tests.

Reply:
Yes, much better. I would also change the `reason` to point to https://github.com/huggingface/diffusers/pull/12456#issuecomment-3424228544.